pandas/tests/plotting/test_datetimelike.py

```diff
@@ -69,6 +70,31 @@ def test_fontsize_set_correctly(self):
         for label in ax.get_xticklabels() + ax.get_yticklabels():
             assert label.get_fontsize() == 2
 
+    @pytest.mark.parametrize("freq", ["20s","3s","60min","5min","7h","4D","8W","11M","2Q","Y", "3Y", "10Y"])
+    def test_period_range_content(self, freq):
+        # Setup your DataFrame as in your scenario
```

Comment:
instead of the comments, e.g. `# Setup your DataFrame as in your scenario`, could you please just add the number of GitHub issue: `# GH#57594`